packages/tiptap/src/utils/custom-extensions.ts

```diff
@@ -0,0 +1,152 @@
+import { Extension } from '@tiptap/core';
+import BulletList from '@tiptap/extension-bullet-list';
+import Heading from '@tiptap/extension-heading';
+import Image from '@tiptap/extension-image';
+import Link from '@tiptap/extension-link';
+import OrderedList from '@tiptap/extension-ordered-list';
+import TextAlign from '@tiptap/extension-text-align';
+
+const CustomHeading = Heading.extend({
+  levels: [1, 2, 3],
+  addAttributes() {
+    return {
+      level: {
+        parseHTML: (element) => Number(element.tagName.charAt(1)),
+      },
+      class: {
+        default: null,
+        rendered: false,
+        parseHTML: (element) => element.getAttribute('class'),
+      },
+    };
+  },
+  renderHTML({ node, HTMLAttributes }) {
+    const level = node.attrs.level as number;
```

Comment:
```suggestion
    const level = node.attrs.level.toNumber();
```
`node.attrs.level`가 어떤 타입인지는 파악이 안되지만,
타입 단언보단 내장 API를 사용하는게 타입세이프하고 안전하다고 생각해요

Reply:
`const level = Number.parseInt(node.attrs.level)`로 수정하였습니다!